### PR TITLE
File removal

### PR DESCRIPTION
**Pull request by lsereda**

### CODE REVIEW

File: src/main/resources/templates/preview_code.html [hunk 1]
```diff
@@ -25,13 +25,38 @@
...
+                        </div>
+                        <div class="modal-body">Are you sure you want to proceed?</div>
+                        <div class="modal-footer">
+                            <button type="button" class="btn btn-secondary" data-dismiss="modal">No, close this window</button>
+                            <a href="#" th:href="@{${delete}}" class="btn btn-secondary">
+                                <i class="fas fa-trash"></i>    Yes, <strong>delete     </strong></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (SzymonSmykala) [inline src/main/resources/templates/preview_code.html:51]: Why you are pasting the same code in 5 files?

File: src/main/kotlin/pl/edu/uj/ii/ksi/mordor/controllers/FilesystemController.kt [hunk 2]
```diff
@@ -171,6 +177,24 @@ class FilesystemController(
         response.flushBuffer()
     }
 
+    @Secured(Permission.ACCESS_ADMIN_PANEL_STR)
+    @GetMapping("/delete/**")
+    fun delete(request: HttpServletRequest, response: HttpServletResponse): ModelAndView {
+        val path = request.servletPath.removePrefix("/delete/")
+        val entity = repoService.getEntity(path) ?: throw NotFoundException(path)
+
```
Comment (apardyl) [inline src/main/kotlin/pl/edu/uj/ii/ksi/mordor/controllers/FilesystemController.kt:185]: Move delete logic to a service

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -53,4 +53,4 @@ mordor.list_hidden_files=false
 # Development settings
 #spring.thymeleaf.cache=false
 #mordor.enable_dev_data_loader=true
-#logging.level.org.springframework.security=DEBUG
+#logging.level.org.springframework.security=DEBUG
```
Comment (apardyl) [inline src/main/resources/application.properties:56]: Revert.

File: src/main/resources/templates/preview_code.html [hunk 4]
```diff
@@ -25,13 +25,38 @@
...
         </div>
         <pre class="card-body p-0 m-0"><code th:text="${text}">
         import test
 
         print("This is a sample code 1234")
         </code></pre>
     </div>
+
```
Comment (apardyl) [inline src/main/resources/templates/preview_code.html:59]: Revert

File: src/main/resources/templates/tree.html [hunk 5]
```diff
@@ -12,6 +12,31 @@
             <span th:if="${itrStat.last}" th:text="${dir.name}">current directory</span>
         </li>
     </ol>
+    <div class="card-header" sec:authorize="hasRole('ROLE_ACCESS_ADMIN_PANEL')">
```
Comment (apardyl) [inline src/main/resources/templates/tree.html:15]: Move to a component.

File: src/main/resources/templates/preview_too_large.html [hunk 6]
```diff
@@ -17,6 +17,30 @@
     <div class="card mt-2">
         <div class="card-header">
             <a href="#" th:href="@{${download}}" class="btn btn-secondary"><i class="fas fa-download"></i> Download</a>
+            <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#exampleModal"
```
Comment (apardyl) [inline src/main/resources/templates/preview_too_large.html:20]: Move to a component.

File: src/main/resources/templates/preview_page.html [hunk 7]
```diff
@@ -39,6 +39,30 @@
     <div class="card mt-2">
         <div class="card-header">
             <a href="#" th:href="@{${download}}" class="btn btn-secondary"><i class="fas fa-download"></i> Download</a>
+            <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#exampleModal"
```
Comment (apardyl) [inline src/main/resources/templates/preview_page.html:42]: Move to a component.

File: src/main/resources/templates/preview_image.html [hunk 8]
```diff
@@ -17,6 +17,30 @@
     <div class="card mt-2">
         <div class="card-header">
             <a href="#" th:href="@{${download}}" class="btn btn-secondary"><i class="fas fa-download"></i> Download</a>
+            <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#exampleModal"
```
Comment (apardyl) [inline src/main/resources/templates/preview_image.html:20]: Move to a component.

**Review by apardyl — CHANGES_REQUESTED** [review]

**Comment by kkiermasz** [comment]
@lsereda ping

**Comment by kkiermasz** [comment]
I close this request due to @lsereda inactivity.

**Pull request closed without merging**